src/views/Home.vue

```diff
@@ -1,53 +1,53 @@
 <template>
-    <v-container class="home pt-0" fluid>
+    <v-container class="home pt-4" fluid>
         <LoadingOverlay :isLoading="isLoading" :showError="hasError" />
-        <v-row v-show="!isLoading && !hasError">
-            <v-col>
-                <v-row class="d-flex justify-space-between px-3 pb-3 pt-1">
-                    <div class="text-h6">
-                        {{ $t("views.home.liveOrUpcomingHeading") }}
-                    </div>
-                    <v-btn icon @click="currentGridSize = (currentGridSize + 1) % 3" v-if="!$store.state.isMobile">
-                        <v-icon>{{ $store.getters.gridIcon }}</v-icon>
-                    </v-btn>
-                </v-row>
-                <VideoCardList
-                    :videos="live"
-                    includeChannel
-                    includeAvatar
-                    :limitRows="2"
-                    :cols="colSizes"
-                    :dense="currentGridSize > 0"
-                >
-                </VideoCardList>
-                <v-divider class="my-3" />
-                <v-row class="d-flex justify-space-between px-3 pt-1 pb-3">
-                    <div class="text-h6">
-                        {{ $t("views.home.recentVideosHeading") }}
-                    </div>
-                    <v-btn-toggle v-model="recentVideoFilter" mandatory dense>
-                        <v-btn value="all">
-                            {{ $t("views.home.recentVideoToggles.all") }}
-                        </v-btn>
-                        <v-btn value="stream">
-                            {{ $t("views.home.recentVideoToggles.official") }}
-                        </v-btn>
-                        <v-btn value="clip">
-                            {{ $t("views.home.recentVideoToggles.subber") }}
-                        </v-btn>
-                    </v-btn-toggle>
-                </v-row>
-                <VideoCardList
-                    :videos="videos"
-                    includeChannel
-                    infiniteLoad
-                    @infinite="loadNext"
-                    :infiniteId="infiniteId"
-                    :cols="colSizes"
-                    :dense="currentGridSize > 0"
-                ></VideoCardList>
-            </v-col>
+        <!-- <v-row v-show="!isLoading && !hasError"> -->
```

Comment:
not sure if there's a reason to comment this out, but I would leave it in because of layout shifts

Reply:
okay replaced with a template v-if and the CLS is fixed :D